railgun/test/integration/suite_test.go

```diff
@@ -106,9 +110,6 @@ func deployControllers(ctx context.Context, ready chan ktfkind.ProxyReadinessEve
 		event := <-ready
```

Comment:
note to self: add log prints for stages of initialization of the test environment

Reply:
https://github.com/Kong/kubernetes-ingress-controller/issues/1181